channel-sdk/src/main/java/com/zello/channel/sdk/image/ImageUtils.kt

```diff
@@ -0,0 +1,84 @@
+package com.zello.channel.sdk.image
+
+import android.graphics.Bitmap
+import java.io.ByteArrayOutputStream
+
+/// The dimensions of the bitmap. Packages the width and height into a single object.
+internal val Bitmap.dimensions: Dimensions
+	get() {
+		return Dimensions(width, height)
+	}
+
+internal data class Dimensions(val width: Int, val height: Int) {
+	companion object {
+		fun square(dimension: Int) = Dimensions(dimension, dimension)
+	}
+}
+
+internal object ImageUtils {
```

Comment:
I guess this is only a stylistic difference, but I think the methods in here would work really well as extensions to `Bitmap` rather than as part of an `object`

Reply:
Good point. I'm hesitant to use extension methods because it makes it harder for me to understand code I'm not familiar with, but this project is small enough that it should be okay.